packages/expo-contacts/CHANGELOG.md

```diff
@@ -5,7 +5,7 @@
 - Fixed an issue where the `requestPermissionsAsync` promise throws when denying access to contacts on iOS. ([#29529](https://github.com/expo/expo/pull/29529) by [@jp1987](https://github.com/jp1987))
 - Fixed an issue where the `presentFormAsync` promise doesn't resolve when the form is closed on Android. ([#29201](https://github.com/expo/expo/pull/29201) by [@jp1987](https://github.com/jp1987))
 - Fixed an issue where the `presentContactPickerAsync` promise doesn't resolve when using the Android back button. ([#29202](https://github.com/expo/expo/pull/29202) by [@jp1987](https://github.com/jp1987))
-
+- Fixed an issue with Xcode 16 Beta builds. ([#29907](https://github.com/expo/expo/pull/29907) by [@alantoa](https://github.com/alantoa))
```

Comment:
Oops, looks like the changelog was messed up with the entries above – they should be in "bug fixes" section 😅  Could you move them there (including your entry)?

Reply:
ah yeah, sorry. just moved it.